config/200-serviceaccount.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cloud-run-events-controller
+  namespace: cloud-run-events
+  labels:
+    events.cloud.run/release: devel
+secrets:
+- name: cloud-run-events-source-key
```

Comment:
For my own edification, does this mean this ServiceAccount has access to this Secret?

Reply:
good question. I do not know. This was moved from knative.